### PR TITLE
feat. added card

### PR DESCRIPTION
**Pull request by hiteshverma27**
Added card with all variations.

### CODE REVIEW

File: Styles/card.css [hunk 1]
```diff
@@ -0,0 +1,148 @@
+.card-v{
+    display: flex;
+    flex-direction: column;
```
Comment (aniketsingh13) [inline Styles/card.css:3]: use utility class for this. if it is repeat more in your code

File: Styles/card.css [hunk 2]
```diff
@@ -0,0 +1,148 @@
...
+    justify-content: space-between;
+    height: 32rem;
+    width: 18rem;
+    background-color: white;
+    padding: 0.5rem;
```
Comment (aniketsingh13) [inline Styles/card.css:8]: same for padding utility

File: Styles/card.css [hunk 3]
```diff
@@ -0,0 +1,148 @@
...
+    background-color: var(--btn-color-confirm);
+    color: white;
+}
+.card-v button:hover{
+    background-color: var(--btn-color-confirm-hover);
+}
+.card-rating{
+    background-color: #388E3C;
+    color: white;
+    display: flex;
```
Comment (aniketsingh13) [inline Styles/card.css:25]: same here

**Review by aniketsingh13 — APPROVED** [review]
use utility and other thing look good to me

File: Styles/card.css (hunk 1 above)
Comment (hiteshverma27) [reply Styles/card.css:3]: I want the user to mention only one class for eg. card to make a component card and furthermore I also made utility classes and you can checkout it in Utility folder.

**Review by surbhi84 — APPROVED** [review]
Looks good to me.

File: Pages/card.html [hunk 4]
```diff
@@ -0,0 +1,272 @@
...
+    <link
+      href="https://fonts.googleapis.com/icon?family=Material+Icons"
+      rel="stylesheet"
+    />
+    <script src="/JS/toggle.js" defer></script>
+    <link
+      href="https://fonts.googleapis.com/icon?family=Material+Icons+Outlined"
+      rel="stylesheet"
+    />
+    <link
```
Comment (viru2001) [inline Pages/card.html:21]: you should import this in your main CSS file and so you don't need to import it again in HTML

File: Pages/card.html [hunk 5]
```diff
@@ -0,0 +1,272 @@
...
+            </svg>
+          </a>
+          <h2 class="logo-heading">Orca UI</h2>
+        </div>
+        <div class="nav-links mx-5">
+          <a
+            class="mx-3 nav-link-items py-2 px- font-600"
+            href="/Pages/installation.html"
+            >Installation</a
+          >
```
Comment (viru2001) [inline Pages/card.html:53]: `px-` ? is this a mistake ?

File: Styles/OrcaUI-styles.css [hunk 6]
```diff
@@ -16,7 +16,6 @@ body {
     "d c c c c"
     "d fn fn fn fn"
     "d f f f f";
```
Comment (viru2001) [inline Styles/OrcaUI-styles.css:18]: you should give proper name to grid template areas

**Review by viru2001 — COMMENTED** [review]
UI is good . I gave few suggestions. Rest everything is good  .

**Review by yogesh- — APPROVED** [review]
looks fine

File: Pages/card.html (hunk 5 above)
Comment (hiteshverma27) [reply Pages/card.html:53]: yes, I will fix that

File: Styles/OrcaUI-styles.css (hunk 6 above)
Comment (hiteshverma27) [reply Styles/OrcaUI-styles.css:18]: sure

**Pull request merged**